alf/algorithms/agent.py

```diff
@@ -127,6 +127,7 @@ def __init__(self,
         rl_algorithm = rl_algorithm_cls(
             observation_spec=rl_observation_spec,
             action_spec=action_spec,
+            config=config,
```

Comment:
Why do we need config here? usually passing a config means that this algorithm needs to create a replay buffer or set rollout state stuff.

Reply:
Good catch, this is only meant for updating in unroll, where a separate replay buffer for unroll experience is created in DdpgAlgorithm.

Reply:
I'm still confused. If only config is passed but not env, then no replay buffer will be created?